Migrate dropdowns in admin from Foundation to Tailwind

andreslucena opened the pull request:
#### :tophat: What? Why?

With the introduction of the Redesign in v0.28, we changed some of the Dropdowns components to use a new approach, with more Accessibility support:

- https://github.com/jonathanlevaillant/a11y-dropdown-component + TailwindCSS styles

The problem is that "some" of the dropdown buttons in the admin panel still were using the old legacy approach ([Foundation Dropdown Menu](https://get.foundation/sites/docs/dropdown-menu.html)).

As we're planning to introduce new Dropdowns in the near future (see https://github.com/decidim/decidim/issues/14651), I think it's a good time to pay this technical debt and be one day closer to fully removing Foundation. 

#### :pushpin: Related Issues
 
- Related to https://github.com/decidim/decidim/issues/14651
 

#### Testing

- All the dropdowns should work as before.
- The CI should pass
- Pages to check: TBD

### :camera: Screenshots

![image](https://github.com/user-attachments/assets/714d5b0d-da10-41e7-95cd-8fe454ffc8b8)

:hearts: Thank you!

andreslucena commented:
@alecslupu this is ready for the review. @carolromero and @julietapasetti already checked this out, and we need this to move forward with #14651

Thanks!

alecslupu reviewed (changes requested):
The Drop down buttons are slightly higher than the regular buttons: 

This branch: 
![image](https://github.com/user-attachments/assets/a252ee15-60dd-4063-b7ac-0afdec1ade51)

VS nighly: 
![image](https://github.com/user-attachments/assets/a1204153-c02f-4f56-8015-71241e570760)


In the previous implementation we had a hover effect, which now is missing (the hovered element is not being colored)

![image](https://github.com/user-attachments/assets/309627c3-3e67-4e99-8cf3-1a4a8babdd4a)

Nightly:
![image](https://github.com/user-attachments/assets/12a4b9b1-e11e-49fc-80dc-f38dbb1f01ee)

andreslucena commented:
> The Drop down buttons are slightly higher than the regular buttons:

It was actually a problem with the svg icons size that was making the button a bit bigger.  Fixed on a1cc5a527cc684813d0c5f15696137475b570b12 

![image](https://github.com/user-attachments/assets/28d0cfca-2370-44c4-bbd9-25f51d5de255)

> In the previous implementation we had a hover effect, which now is missing (the hovered element is not being colored)

Fixed on f391121e10467fdbb0cec83a3ef159f02cc60ed9

![image](https://github.com/user-attachments/assets/dda0d837-53d8-4905-89c6-cdf32cc6cd57)


This is ready for another round!

alecslupu reviewed (changes requested):
There are still errors: 

![image](https://github.com/user-attachments/assets/f6b78999-c40f-45c5-b912-abcf3b1187f2)

![image](https://github.com/user-attachments/assets/31c1a8fc-8c53-49ba-b5da-f3bc0821efb9)

![image](https://github.com/user-attachments/assets/c7cd6eae-390f-445e-91b6-598b6e7a11bf)

andreslucena commented on decidim-admin/app/views/decidim/admin/shared/landing_page/_content_blocks.html.erb:
Fixed at 1dc83ee46c5bb59b982661472b0280c7584dfd52

andreslucena commented on decidim-budgets/app/views/decidim/budgets/admin/projects/_bulk-actions.html.erb:
Fixed at 05182033e2022eb996eac9c5c1ffdcadeb532b0e.

andreslucena commented on decidim-accountability/app/views/decidim/accountability/admin/results/index.html.erb:
Fixed at 05182033e2022eb996eac9c5c1ffdcadeb532b0e.

andreslucena commented on decidim-debates/app/views/decidim/debates/admin/debates/index.html.erb:
Fixed at 05182033e2022eb996eac9c5c1ffdcadeb532b0e.

andreslucena commented on decidim-initiatives/app/views/decidim/initiatives/admin/exports/_dropdown.html.erb:
Fixed at 05182033e2022eb996eac9c5c1ffdcadeb532b0e.

andreslucena commented on decidim-meetings/app/views/decidim/meetings/admin/registrations/edit.html.erb:
Fixed at 05182033e2022eb996eac9c5c1ffdcadeb532b0e.

andreslucena commented on decidim-meetings/app/views/decidim/meetings/admin/registrations_attendees/index.html.erb:
Fixed at 05182033e2022eb996eac9c5c1ffdcadeb532b0e.

alecslupu reviewed (approved):
:+1: 

Merging with codecov failing pipeline.
